nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -353,6 +353,9 @@ The pre-existing [services.ankisyncd](#opt-services.ankisyncd.enable) has been m
 - New options were added to the dnsdist module to enable and configure a DNSCrypt endpoint (see `services.dnsdist.dnscrypt.enable`, etc.).
   The module can generate the DNSCrypt provider key pair, certificates and also performs their rotation automatically with no downtime.
 
+- The ranger file manager was updated from the latest release (which is a few
```

Comment:
Should we even add a changelog for this? Package bumps without consequences to config or state files don't seem worth mentioning. What's the official policy on this?

Reply:
I've seen this done for package that may contain breaking changes. I'd say switching from a tagged release to tracking master is potentially a breaking change, but I'm neutral about this.

Reply:
As Ranger maintainer, I disagree. We pay a lot of attention to backwards compatibility and there haven't been big changes.

Reply:
It's less about compatibility and more about how close master is to production-quality. 100% fair though, I haven't really run into any regressions myself over a few months, so I'm fine with dropping the changelog entry.